kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -392,6 +392,40 @@ struct GetCategoryEarliestStale {
   }
 };
 
+struct GetStaleKeysSummary {
+  const bool read_only = true;
+  std::string description() const {
+    return "getStaleKeysSummary [BLOCK-VERSION-TO]\n"
```

Comment:
We can add a note to the description that mentions to users that this is essentially a linear search and it might take a lot of time.